.github/workflows/upload-wheels.yaml

```diff
@@ -0,0 +1,96 @@
+name: Build
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+
+
+env:
+  CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
+  MACOSX_DEPLOYMENT_TARGET: '10.13'
+  CIBW_BUILD_VERBOSITY: 2
+  # Only build on Python 3.x
+  CIBW_BUILD: 'cp3?-*'
+  CIBW_SKIP: 'cp35-* cp39-* *-manylinux_i686'
+  CIBW_BEFORE_TEST: pip install nose mock
+  CIBW_TEST_COMMAND: nosetests -s -v -P {project}/python/tests
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, macos-latest]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: 'true'
+
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.7'
+
+      - name: Install cibuildwheel
+        run: |
+          python -m pip install cibuildwheel==1.6.4
+      - name: Build wheels on Linux
+        if: runner.os == 'Linux'
+        env:
+          CIBW_BEFORE_BUILD: 'yum update -y; yum -y install hdf5-devel'
+        run: |
+          python -m cibuildwheel --output-dir dist
+      - name: Build wheels Mac OS
+        if: runner.os == 'macOS'
+        env:
+          CIBW_BEFORE_BUILD: 'brew update; brew install cmake hdf5'
+        run: |
+          python -m cibuildwheel --output-dir dist
+      - name: Store wheel as artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: dist
+          path: dist
+
+  build_sdist:
```

Comment:
if we keep the `python_push_sdist.sh` file, it makes things easier to test, imo, and move away to another CI system at somepoint.  Does adding it in the action make something easier?